setup.cfg

```diff
@@ -27,6 +27,7 @@ install_requires =
     Click
     sqlparse
     colorama
```

Comment:
You can make this requirement apply only for py<=3.6
https://www.python.org/dev/peps/pep-0508/